app/components/agenda/modal.js

```diff
@@ -83,7 +83,7 @@ export default class AgendaModalComponent extends Component {
       this.toBeDeleted.push(this.currentlyEditing);
       this.isNew = false;
     } else {
-      this.currentlyEditing.rollbackAttributes();
+      // this.currentlyEditing.rollbackAttributes();
```

Comment:
As I said to arne, I think we should remove the commented code, in order to clean the repo :)

Reply:
I am not sure about this line thats why I left it, rollBackAttributes behaviour is wierd when an empty agendapoint is created. but technically this should be here